.travis.yml

```diff
@@ -8,6 +8,19 @@ notifications:
 node_js:
 - '10'
 install:
-  - npm ci
+- npm ci
 script:
-  - npm run travis:verify
+- npm run travis:verify
+before_install:
+- openssl aes-256-cbc -K $encrypted_8e579b44016d_key -iv $encrypted_8e579b44016d_iv -in deploy_key.enc -out deploy_key -d
+
+- chmod 600 deploy_key
+- eval `ssh-agent -s`
+- ssh-add deploy_key
+after_success:
+- .travis/after_success.sh
+env:
+  global:
+  - REPO="git@github.com:RedHatInsights/insights-deployment-test-build"
```

Comment:
Are we updating this to `insights-dashboard-build`?

Reply:
uhwhoops

Reply:
actually I don't think these variables are used any longer. I'm removing them.

Reply:
They're referenced in the `.sh` files, so don't remove :)